libchisel/src/dropsection.rs

```diff
@@ -80,6 +80,7 @@ impl<'a> ModuleTranslator for DropSection {
 mod tests {
     use super::*;
```

Comment:
If we're following the rule of
```
std imports
extern imports
local imports
```
then this goes at the bottom, no?

Reply:
I didn't care about tests.